sys/dev/buslogic/bt_pci.c

```diff
@@ -56,16 +56,11 @@ __FBSDID("$FreeBSD$");
 #define PCI_DEVICE_ID_BUSLOGIC_MULTIMASTER_NC	0x0140104Bul
 #define PCI_DEVICE_ID_BUSLOGIC_FLASHPOINT	0x8130104Bul
 
-static struct bt_dev {
-	uint16_t vendorid;
-	uint32_t deviceid;
-	const char *description;
-} bt_devs[] = {
-	{0x104B, PCI_DEVICE_ID_BUSLOGIC_MULTIMASTER,
-	    "Buslogic Multi-Master SCSI Host Adapter"},
-	{0x104B, PCI_DEVICE_ID_BUSLOGIC_MULTIMASTER_NC,
-	    "Buslogic Multi-Master SCSI Host Adapter"},
-	{0, 0, 0},
+struct pci_device_table bt_devs[] = {
+	{PCI_DEV(0x104B, 0x1040),
```

Comment:
These should use symbolic stuff. Maybe I need a new macro PCI_DEVID :)

Reply:
Is this the macro formulation that we need here ?
#define PCI_DEVID(did)							\
	PCI_DEV((uint16_t)did, (uint16_t)(did >> 16))